articles/batch/batch-upgrade-policy.md

```diff
@@ -0,0 +1,223 @@
+> - For more information, see [Supplemental Terms of Use for Microsoft Azure Previews](https://azure.microsoft.com/support/legal/preview-supplemental-terms/).
+
+When you create an Azure Batch pool, you can provision the pool with nodes that have Auto OS Upgrade enabled. This article explains how to set up a Batch pool with Auto OS Upgrade.
+
+## Why use Auto OS Upgrade?
+
+Auto OS Upgrade is used to implement an automatic operating system upgrade strategy and control within Azure Batch Pools. Here are some reasons for using Auto OS Upgrade:
+
+- **Security.** Auto OS Upgrade ensures timely patching of vulnerabilities and security issues within the operating system image, thereby enhancing the security of compute resources. This helps prevent potential security vulnerabilities from posing a threat to applications and data.
+- **Minimized Availability Disruption.** Auto OS Upgrade is used to minimize the availability disruption of compute nodes during OS upgrades. This is achieved through task-scheduling-aware upgrade deferral and support for rolling upgrades, ensuring that workloads experience minimal disruption.
+- **Flexibility.** Auto OS Upgrade allows you to configure your automatic operating system upgrade strategy, including percentage-based upgrade coordination and rollback support. This means you can customize your upgrade strategy to meet your specific performance and availability requirements.
+- **Control.** Auto OS Upgrade provides you with control over your operating system upgrade strategy to ensure secure, workload-aware upgrade deployments. You can tailor your policy configurations to meet the specific needs of your organization.
+
+In summary, the use of Auto OS Upgrade helps you achieve automatic operating system upgrades, leading to improved security, minimized availability disruptions, and greater control and flexibility for your workloads. This is crucial for maintaining the health and security of compute resources.
+
+## How does Auto OS Upgrade work?
+
+When upgrading images, VMs in Azure Batch Pool will follow roughly the same work flow as VirtualMachineScaleSets. To learn more about the detailed steps involved in the Auto OS Upgrade process for VirtualMachineScaleSets, you can refer to [VMSS page](../virtual-machine-scale-sets/virtual-machine-scale-sets-automatic-upgrade.md#how-does-automatic-os-image-upgrade-work).
+
+However, there will be a difference for Batch to handle upgrades if *automaticOSUpgradePolicy.osRollingUpgradeDeferral* is set to 'true'. When an upgrade becomes available while a batch node is actively running a task, this property will postpone OS upgrades on the node. Once the node transitions to an idle state, Batch will issue a callback and initiate the upgrade process. Thus, We strongly advise enabling the *automaticOSUpgradePolicy.osRollingUpgradeDeferral* feature.
```

Comment:
"However,  if *automaticOSUpgradePolicy.osRollingUpgradeDeferral* is set to 'true', when an upgrade becomes available while a batch node is actively running tasks the upgrade will be delayed until all tasks have been completed on the node."